### PR TITLE
Re-added Overview to components

### PR DESCRIPTION
**Pull request by erikeh**
Added an Overview component as a test

### CODE REVIEW

**Review by Lordnickim — COMMENTED** [review]
Lols I can see the Changes.

**Review by Lordnickim — APPROVED** [review]
First Approval

File: client/src/components/Overview.jsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const OverView = () => (
+  <h1>This is the overview, goodbye.</h1>
```
Comment (raphaelspies) [inline client/src/components/Overview.jsx:4]: I see you used the word "overview". You should use the word "underview" from now on; "overview" is deprecated as of ES12.

**Review by raphaelspies — CHANGES_REQUESTED** [review]

File: client/src/components/Overview.jsx (hunk 1 above)
Comment (Lordnickim) [reply client/src/components/Overview.jsx:4]: I second this.

**Review by Lordnickim — CHANGES_REQUESTED** [review]

**Review by Lordnickim — APPROVED** [review]

File: client/src/components/Overview.jsx (hunk 1 above)
Comment (raphaelspies) [reply client/src/components/Overview.jsx:4]: Sorry-- still waiting for "underview" update. See comment above

**Review by raphaelspies — CHANGES_REQUESTED** [review]

**Review by raphaelspies — APPROVED** [review]
I have no idea what this code is doing. LGTM!

**Pull request merged**